README.md

```diff
@@ -1,6 +1,18 @@
-# RemoteRecord
+![RemoteRecord: Ready-made remote resource structures.](doc/header.svg)
 
-Ready-made remote resource structures.
+---
+
+![Remote Record](https://github.com/raisedevs/remote_record/workflows/Remote%20Record/badge.svg)
+[![Gem Version](https://badge.fury.io/rb/remote_record.svg)](https://badge.fury.io/rb/remote_record)
+
+Every API speaks a different language. Maybe it's REST, maybe it's SOAP, maybe
+it's GraphQL. Maybe it's got its own Ruby client, or maybe you need to roll your
+own. But what if you could just pretend it existed in your database?
+
+RemoteRecord lets you store remote resources by ID and have them auto-populated
+whenever you want to read from them. Whether you're dealing with a user on
+GitHub, a track on Spotify, a place on Google Maps, or a resource on your
+internal infrastructure, you can use RemoteRecord to wrap fetching it.
```

Comment:
This looks good. I may emphasize the consistent interface aspect more, IE:

RemoteRecord provides a consistent ActiveRecord inspired interface for all of your application's APIs.